cmd/main.go

```diff
@@ -62,3 +64,27 @@ func echoCheckIdHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	fmt.Fprintf(w, "ID：%v\nName:%v", idInt, name)
 }
+
+func echoTimeHandler(w http.ResponseWriter, r *http.Request) {
+	query := r.URL.RawQuery
+	t := time.Now()
+
+	jst, err := time.LoadLocation("Asia/Tokyo")
+	if err != nil {
+		fmt.Fprintf(w, "Time：%s\n", err)
+		return
+	}
+	jst_time := t.In(jst)
+
+	if query == "date" {
+		const dateFormat = "2006/01/02"
+		fmt.Fprintf(w, "Time：%s\n", jst_time.Format(dateFormat))
+		return
+	} else if query == "time" {
+		const timeFormat = "15:04:05"
+		fmt.Fprintf(w, "Time：%s\n", jst_time.Format(timeFormat))
+		return
+	}
+
+	fmt.Fprintf(w, "Time：%s\n", jst_time)
```

Comment:
こういう場合はこうかな。
ResponseWriterへの書き込みは一箇所に絞ったほうがいいと思うのです。
＆こうしておくと、ロジックを関数へ切り出すのも容易になるかと思います。

```suggestion
	var ret string
	switch query {
	case: "date":
		const dateFormat = "2006/01/02"
		ret = fmt.Sprintf(w, "Time：%s\n", jst_time.Format(dateFormat))
	case "time": 
		const timeFormat = "15:04:05"
		ret = fmt.Sprintf(w, "Time：%s\n", jst_time.Format(timeFormat))
	}

	fmt.Fprintf(w, ret)
```

Reply:
あと、const はなるべくfile先頭へ。
これならimportのすぐ下かな。
pkg単位でスコープきられるので、まとめて定義しておかないと名前衝突でムキーとなりがちです。